packages/extension/src/e2e.test.js

```diff
@@ -72,6 +72,30 @@ test(
   TEST_TIMEOUT
 )
 
+// TODO need a good indicator that deeplinking is working
+test(
+  'newtab page should be able to deeplink into a exercisee',
+  async () => {
+    await page.goto('chrome://newtab?play=true')
+    await waitDOMLoaded()
+    await wait(500)
+    const el = await page.$('[data-testid="breath-player--count"]')
+    expect(el).not.toBe(null)
+
+    const initialCount = await (await el.getProperty(
+      'textContent'
+    )).jsonValue()
+    expect(initialCount.trim()).toBe('5')
+
+    await wait(7000)
```

Comment:
Not super happy about this, debating if I want to be able to override the timing of the breath to be able to make this faster.